src/main/resources/controllers-context.xml

```diff
@@ -11,7 +11,7 @@
        https://www.springframework.org/schema/context/spring-context.xsd">
 
     <bean class="com.example.dog.controller.DogController">
-        <constructor-arg ref="jdbcDogDao"/>
+        <constructor-arg ref="dogService"/>
```

Comment:
You can use autowiring by type (is set in the attribute on the root xml element) and not specify parameters explicitly.